### PR TITLE
chore: update workflows from templates

### PR DESCRIPTION
**Pull request by skjnldsv**
This PR was automatically generated by the workflow dispatcher
Please check that there are no reverted changes before merging ⚠

See templates in https://github.com/nextcloud/.github

### CODE REVIEW

File: .github/workflows/dependabot-approve-merge.yml [hunk 1]
```diff
@@ -31,20 +31,13 @@ jobs:
...
     steps:
-      - uses: mdecoleman/pr-branch-name@bab4c71506bcd299fb350af63bb8e53f2940a599 # v2.0.0
-        id: branchname
-        with:
-          repo-token: ${{ secrets.GITHUB_TOKEN }}
-
-      # Github actions bot approve
+      # GitHub actions bot approve
       - uses: hmarr/auto-approve-action@b40d6c9ed2fa10c9a2749eca7eb004418a705501 # v2
-        if: contains(steps.branchname.outputs.branch, '/webrtc-adapter-') != true && contains(steps.branchname.outputs.branch, '/nextcloud/vue-') != true
```
Comment (Antreesy) [inline .github/workflows/dependabot-approve-merge.yml:41]: cc @nickvergessen 
this was intended from our side, IIRC

Comment (nickvergessen) [reply .github/workflows/dependabot-approve-merge.yml:41]: yeah, just revert it and push

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**